plugins/extensions/plugins/RocketsPlugin.h

```diff
@@ -179,6 +194,24 @@ class RocketsPlugin : public ExtensionPlugin
     ::brayns::v1::Scene _remoteScene;
     ::brayns::v1::ForceRendering _remoteForceRendering;
     ::brayns::v1::CircuitConfigurationBuilder _remoteCircuitConfigBuilder;
+
+    class Timer
```

Comment:
Can I ask what's the purpose of this timer?

Reply:
To limit the image stream to the configurable image stream FPS. 60 FPS by default